slsa-framework/slsa-verifier
# docs: update docs for release v2.0.1

This PR
* Updates the SHA256SUM.md for the releaes v2.0.1
* Updates the README.md with the v2 module path and adds a known issue for 2.0.1 panic.

To verify hash

1. Download the binary and provenance from https://github.com/slsa-framework/slsa-verifier/releases/tag/v2.0.1
2. Clone the slsa-verifier repo, compile and verify the provenance:
```
$ git clone git@github.com:slsa-framework/slsa-verifier.git
$ cd slsa-verifier
$ (Optional: git checkout tags/v2.0.1)
$ go run ./cli/slsa-verifier verify-artifact ~/Downloads/slsa-verifier-linux-amd64 --provenance-path ~/Downloads/slsa-verifier-linux-amd64.intoto.jsonl --source-uri github.com/slsa-framework/slsa-verifier --source-tag v2.0.1
```
3. Get the hash.
Either:
```
cat slsa-verifier-linux-amd64.intoto.jsonl | jq -r '.payload' | base64 -d | jq -r '.subject[0].digest.sha256'
```
or

```
sha256sum slsa-verifier-linux-amd64
```

Signed-off-by: Asra Ali <asraa@google.com>
